test/plugins/test_ecs_release.py

```diff
@@ -114,7 +118,12 @@ def test_builds_container(self, fixtures):
             assert plugin_data == {'image_id': image_name}
 
             check_call.assert_called_once_with(
-                ['docker', 'build', '-t', image_name, '.']
+                [
```

Comment:
Cosmetic comment - I'd pull the `[` next to the `(` making it `([` [...] `])`

Reply:
👍